src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -461,6 +474,14 @@ static boolean isVariantInCodingRegion(final GencodeFuncotation.VariantClassific
 
             // Only annotate on the `basic` transcripts:
             if ( isBasic ) {
+
+                // Make sure that we have the transcript in our list.
+                // If we don't, warn the user and do not add any funcotations:
+                if ( !transcriptIdMap.containsKey(transcript.getTranscriptId()) ) {
```

Comment:
Does this actually ever happen?

Reply:
Yes - this happens in the case where you have a subset of the GENCODE GTF file and you try to annotate something outside that genetic region.

Reply:
I ran into this issue with my unit tests.